src/Pages/ProductDetail/ProductDetailRight/ShoeColor/ShoeColor.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import "./ShoeColor.scss";
+
+class ShoeColor extends Component {
+  shoeColorArr = [
+    "https://image.converse.co.kr/cmsstatic/product/560250C_560250C_pdp-option.jpg?browse=",
+    "https://image.converse.co.kr/cmsstatic/product/560251C_560251C_pdp-option.jpg?browse=",
+  ];
+
+  listShoeColor = this.shoeColorArr.map((shoe, i) => {
+    if (i === 0) {
```

Comment:
```suggestion
    if (i) {
```
숫자 0은 자바스크립트에서 falsy한 값으로 인식하기 때문에 i만 써줘도 됩니다.